### PR TITLE
feat(user): Generate passwords with an external API [COM-23]

### PR DESCRIPTION
**Pull request by IsaacGzOv**
Request and storage the new password fron an API 
[COM-23]

[COM-23]: https://comunidadesbootcamp.atlassian.net/browse/COM-23?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/services/generatePass-service.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                    });
+                }
+            }
+          });
+    }
+    res.send({
+        status: "OK",
+        rowsModify: data.length
+    });
+};
```
Comment (gonzajo) [inline src/services/generatePass-service.js]: check this library to be able to do so [NPM](https://www.npmjs.com/package/generate-password)
without using an API to generate the password

**Review by gonzajo — CHANGES_REQUESTED** [review]

File: src/repositories/user-repository.js [hunk 2]
```diff
@@ -107,11 +106,24 @@ const findUserByEmail = async (mail) => {
   }
 };
 
+const setPassword = async (userid, password) => {
+    try {
+        const connection = await connect();
+        return connection.query("UPDATE user SET password = ? WHERE user = ?", [
+            password,
```
Comment (gonzajo) [inline src/repositories/user-repository.js]:
```suggestion
        return connection.query("UPDATE user SET password = ? WHERE id = ?", [
            password,
```

File: src/services/generatePass-service.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                status: "FAILED",
+                data: { error: error?.message || error }
+            });
+        }
+    }
+    res.send({
+        status: "OK",
+        rowsModify: data.length
+    });
+};
```
Comment (gonzajo) [inline src/services/generatePass-service.js:28]: asking

**Review by gonzajo — CHANGES_REQUESTED** [review]

File: src/services/generatePass-service.js (hunk 3 above)
Comment (gonzajo) [reply src/services/generatePass-service.js:28]: @isaacGon this will change right?

**Review by gonzajo — COMMENTED** [review]
question
> @isaacGon this will change right?

File: src/services/generatePass-service.js (hunk 3 above)
Comment (IsaacGzOv) [reply src/services/generatePass-service.js:28]: yes, im going to create a new branch to make all the process for the sendmail

**Review by gonzajo — APPROVED** [review]

**Pull request merged**